lib/tapioca/commands/abstract_gem.rb

```diff
@@ -67,7 +67,7 @@ def compile_gem_rbi(gem)
           reason: "types exported from the `#{gem.name}` gem",
         ) if @file_header
 
-        rbi.root = Runtime::Trackers::Autoload.with_disabled_exits do
+        rbi.root = Tapioca.with_disabled_exits do
```

Comment:
I think it could have stayed under `Runtime` with other runtime related helpers

Reply:
We can move both of the runtime helper methods under runtime, no problem.